src/connectors/rating-menu/__tests__/connectRatingMenu-test.js

```diff
@@ -370,116 +395,159 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/rating-menu
     });
   });
 
-  describe('routing', () => {
-    const getInitializedWidget = (config = {}) => {
-      const rendering = jest.fn();
-      const makeWidget = connectRatingMenu(rendering);
-
-      const attribute = 'grade';
-      const widget = makeWidget({
-        attribute,
-        ...config,
+  describe('getWidgetState', () => {
+    test('should give back the object unmodified if the default value is selected', () => {
+      const [widget, helper] = getInitializedWidget();
```

Comment:
Not a big thing at all, but if `getInitializedWidget` returned `{widget, helper, refine}` instead of `[widget, helper, refine]`, it would be easier to destructure it without having to think about the returned order, like
`const { helper, widget } = getInitializedWidget();`.

And `const {widget, refine} = ...` instead of `const [widget, , refine] = ...`.

Reply:
Indeed it's a fair point, we'll revisit this function anyway because we can't even use it right now inside the tests for the `getWidgetSearchParameters`. We can do this change at the same time (the function is not part of the PR, it just moved).